src/native/libs/System.Globalization.Native/pal_calendarData.m

```diff
@@ -11,6 +11,15 @@
 #endif
 
 #if defined(TARGET_MACCATALYST) || defined(TARGET_IOS) || defined(TARGET_TVOS)
+#define GREGORIAN_NAME "gregorian"
+#define JAPANESE_NAME "japanese"
+#define BUDDHIST_NAME "buddhist"
+#define HEBREW_NAME "hebrew"
+#define DANGI_NAME "dangi"
+#define PERSIAN_NAME "persian"
+#define ISLAMIC_NAME "islamic"
+#define ISLAMIC_UMALQURA_NAME "islamic-umalqura"
+#define ROC_NAME "roc"
```

Comment:
Would it work if we had these + the defines in `pal_calendarData.c` in the header file?

Reply:
Moved this to `pal_calendarData.h` as both `pal_calendarData.c` and `pal_calendarData.m` are using it